kyaml/fn/framework/framework.go

```diff
@@ -116,8 +116,21 @@ func Execute(p ResourceListProcessor, rlSource *kio.ByteReadWriter) error {
 	}
 	rl.FunctionConfig = rlSource.FunctionConfig
 
+	// We store the original
+	nodeAnnos, err := kio.StoreInternalAnnotations(rl.Items)
```

Comment:
When I read this line for the first time, `StoreInternalAnn..` reads as if it is storing the annotations internally, but then I realized it is actually returning the mapped internal annotations. The act of storing is being done here itself :) I couldn't come up with better name other than... `GetInternalAnnotations` which is already being used for different purpose I guess.

Reply:
Hmm... Perhaps `StoreFileAnnotations` (analogous to the existing `GetFileAnnotations`) might be a better name